src/components/Layout/Modal/Modal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+import classes from './Modal.module.css';
+
+const Backdrop = (props) => {
+  return <div className={classes.backdrop} onClick={props.onClose} />;
+};
+
+const ModalOverlay = (props) => {
+  return (
+    <div className={classes.modal}>
+      <div className={classes.content}>{props.children}</div>
+    </div>
+  );
+};
```

Comment:
as this are different components its better to have them on their own dedicated file. Also remember the props